hedera-node/src/main/java/com/hedera/services/contracts/gascalculator/GasCalculatorHederaV18.java

```diff
@@ -91,11 +88,7 @@ public Gas logOperationGasCost(
 	}
 
 	protected long ramByteHoursTinyBarsGiven(long consensusTime, HederaFunctionality functionType) {
```

Comment:
nit: Can't we directly call `GasCalculatorHederaUtil.ramByteHoursTinyBarsGiven(usagePrices, exchange, consensusTime, functionType)` inside the `calculateStorageGasNeeded` method and remove the `protected long ramByteHoursTinyBarsGiven(long consensusTime, HederaFunctionality functionType)`